### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#331] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152324437-aa965de7-5f06-4486-992d-6dcd2b164f7b.png)

**Comment by danieldinella** [comment]
A ⟷ B = 1 solo se A = B.
1. Essendo uguali A ∧ B = 1 solo se A = B = 1.
2. Essendo uguali A ∨ B = 1 solo se A = B = 1.
3. Essendo uguali A → B = 1 in ogni caso.
4. Nel caso in cui A = B = 1 possiamo riscrivere la proposizione come C → C mentre nel caso in cui A = B = 0 il risultato sarà 
0 → 0 dunque la proposizione sarà sempre vera.

**Comment by CiottoloMaggico** [comment]
![immagine](https://user-images.githubusercontent.com/79638284/217795663-f311f0fd-bfbc-437c-b0d0-7ea74c2dda76.png)
